packages/language-support/src/autocompletion/completionCoreCompletions.ts

```diff
@@ -282,6 +283,13 @@ function calculateNamespacePrefix(
   return namespacePrefix;
 }
 
+const baseTriggerCharacters = ['.', '(', ')', ','];
+export const snippetTriggerCharacters = ['<', '-'];
```

Comment:
Did you solve the problem with the `-` and `<`?

Reply:
yep, will open a new PR with only the relevant changes